package.json

```diff
@@ -30,5 +30,8 @@
     "node-gyp": "^10.0.1",
     "prettier": "^3.1.1",
     "yarn-deduplicate": "^6.0.2"
+  },
+  "volta": {
+    "node": "20.12.2"
```

Comment:
```suggestion
```

seems not relate to this pr. if we decide to add that we could have another pr to discuss the version enforcing.

Reply:
oh i saw this https://github.com/expo/expo/pull/28167